core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -52,5 +52,7 @@ registry.ci.openshift.org/ocp-kni/4.11:cnf-tests quay.io/openshift-kni/cnf-tests
 registry.ci.openshift.org/ocp-kni/4.11:dpdk quay.io/openshift-kni/dpdk:4.11 quay.io/openshift-kni/dpdk:4.11.0 quay.io/openshift-kni/dpdk:latest
 registry.ci.openshift.org/ocp-kni/4.11:ztp-site-generator quay.io/openshift-kni/ztp-site-generator:4.11 quay.io/openshift-kni/ztp-site-generator:4.11.0 quay.io/openshift-kni/ztp-site-generator:latest
 # cluster-group-upgrades-operator
-registry.ci.openshift.org/ocp-kni/4.10:cluster-group-upgrades-operator quay.io/openshift-kni/cluster-group-upgrades-operator:4.10.0 quay.io/openshift-kni/cluster-group-upgrades-operator:latest
-registry.ci.openshift.org/ocp-kni/4.10:cluster-group-upgrades-operator-bundle quay.io/openshift-kni/cluster-group-upgrades-operator-bundle:4.10.0 quay.io/openshift-kni/cluster-group-upgrades-operator-bundle:latest
+registry.ci.openshift.org/ocp-kni/4.10:cluster-group-upgrades-operator quay.io/openshift-kni/cluster-group-upgrades-operator:4.10.0
+registry.ci.openshift.org/ocp-kni/4.10:cluster-group-upgrades-operator-bundle quay.io/openshift-kni/cluster-group-upgrades-operator-bundle:4.10.0
+registry.ci.openshift.org/ocp-kni/4.10:cluster-group-upgrades-operator quay.io/openshift-kni/cluster-group-upgrades-operator:4.11.0 quay.io/openshift-kni/cluster-group-upgrades-operator:latest
+registry.ci.openshift.org/ocp-kni/4.10:cluster-group-upgrades-operator-bundle quay.io/openshift-kni/cluster-group-upgrades-operator-bundle:4.11.0 quay.io/openshift-kni/cluster-group-upgrades-operator-bundle:latest
```

Comment:
Shouldn't this be 4.11? Same on the line below.

Reply:
Oops, you are right, thx